image-layout.md

```diff
@@ -15,9 +15,12 @@ The image layout has two top level directories:
 - "refs" contains [descriptors][descriptors]. Commonly pointing to an [image manifest](manifest.md#image-manifest) or an [image manifest list](manifest-list.md#oci-image-manifest-list-specification).
 
 
-It also contains a file that is used to identify the layout version:
+The image layout MUST contain an `oci-layout` file:
 
-- "oci-layout" MUST contain a JSON object with a version field `{"imageLayoutVersion": "1.0.0"}` and MAY include additional fields.
+- It MUST be a JSON object
+- It MUST contain an `imageLayoutVersion` field
+    - The `imageLayoutVersion` value will align with the OCI Image Specification version at the time changes to the layout are made, and will pin a given version until changes to the layout are required
```

Comment:
You've gone with “OCI Image Specification”, but the title for this spec is “[Open Container Initiative Image Format Specification][1]”.  I'm fine changing “Open Container Initiative” → “OCI”, but don't think we should be dropping “Format” here unless we also drop it from the spec title.  I don't mind which way we go, but I think we should stay consistent.

I also [still][2] think we should either semantically version `imageLayoutVersion` or point out that this is not semantically versioned.

[1]: https://github.com/opencontainers/image-spec/blob/a061b53a054decf479b006866bbe9f2642b50129/README.md#open-container-initiative-image-format-specification
[2]: https://github.com/opencontainers/image-spec/pull/434#discussion_r85972877